package.json

```diff
@@ -21,73 +21,82 @@
     "test:e2e": "jest --config ./test/jest-e2e.json"
   },
   "dependencies": {
-    "@mikro-orm/core": "^4.3.3",
-    "@mikro-orm/nestjs": "^4.2.0",
-    "@mikro-orm/postgresql": "^4.5.7",
+    "@apollo/gateway": "^2.3.2",
```

Comment:
Is it intended? As far as I know, we don't use apollo, as well as it is not listed in the original package.json

Reply:
Yeah, please extract to separate one